### DISCUSSION TITLE
I wantEdit easing

### DISCUSSION THREAD

**Discussion by tawa-create** [keiwando/toonsquid#628] [Ideas]
I would like to have the ability to freely edit easing!
 I would also like to see some effects(like blur) a feature like AlightMotion would be great.
(Sorry I don’t have confidence on how to use this site. If I’m rude, I’m sorry)

**Comment by kavlor** [comment]
You should read the 'ToonSquid 2.0 Closed Beta' thread.

**Reply by tawa-create** [reply]
WOW‼️‼️‼️‼️I didn’t notice ‼️‼️‼️‼️THANK YOU SO MUCH‼️‼️‼️‼️‼️🙏🙏🙏🙏
I’m really looking forward to it✨✨✨✨
